test/e2e/tkn-e2e.bats

```diff
@@ -18,6 +18,18 @@ source ./test/helper/helper.sh
         --showlog >&3
     assert_success
 
+    # waiting a few seconds before asserting results
+    sleep 15
+
+    run kubectl delete taskrun --all
+    assert_success
+
+    run tkn task start tkn \
+        --use-param-defaults \
+        --skip-optional-workspace \
+        --showlog >&3
```

Comment:
@vdemeester added section in the e2e to use the default script which is `tkn $@`